test/unit/CowOperator/placeOrder.js

```diff
@@ -679,7 +679,12 @@ describe('placeOrder', function () {
   it('emits an OrderPlaced event', async function () {
     const tx = await swapOperator.placeOrder(contractOrder, orderUID);
     const rcp = await tx.wait();
-
-    expect(rcp.events[2].args.order).to.deep.include.members(Object.values(contractOrder));
+    // Remove duplicate values
+    const rcpOrder = rcp.events[2].args.order.slice(0, 12);
```

Comment:
I don't love this solution. It seems with the upgrade the event receipt has duplicate values  and the BigNumber type is different from parseEther(). Open to suggestions for a cleaner approach.